pkg/cli/admin/mustgather/mustgather.go

```diff
@@ -583,6 +529,65 @@ func (o *MustGatherOptions) Run() error {
 	return errors.NewAggregate(errs)
 }
 
+// podProcessor processes Pods created for collecting each must-gather
+func (o *MustGatherOptions) podProcessor(ns string, podChan chan *corev1.Pod, errCh chan error, wg *sync.WaitGroup) {
```

Comment:
Let's stick with `runWorker` and `processNextWorkItem` naming.

Reply:
Done.